server/controllers/notification.js

```diff
@@ -0,0 +1,48 @@
+const asyncHandler = require('express-async-handler');
```

Comment:
Let's base the branch so this isn't being shown - but this PR has been approved so you could just merge it in and this will go away